alf/examples/dyna_actrepeat_sac_bipedalwalker.gin

```diff
@@ -22,4 +21,6 @@ SacAlgorithm.target_entropy=(
 
 TrainerConfig.algorithm_ctor=@DynamicActionRepeatAgent
 TrainerConfig.debug_summaries=True
-TrainerConfig.unroll_length=2
+
+RewardNormalizer.clip_value=1.0
```

Comment:
With 1.0, a lot of reward can be clipped.

Reply:
changed to 5.0. The new curve looks similar to the current one.